blockbot/blockbot.php

```diff
@@ -26,7 +27,23 @@ function blockbot_uninstall() {
 function blockbot_init_1(App $a) {
 	$crawlerDetect = new CrawlerDetect();
 
+	// List of strings of known "good" agents
+	$agents = ['diaspora-connection-tester', 'DiasporaFederation', 'Friendica', '(compatible; zot)',
+		'Micro.blog', 'Mastodon', 'hackney', 'GangGo', 'python/federation', 'GNU social', 'winHttp',
+		'Go-http-client', 'Mr.4x3 Powered', 'Test Certificate Info', 'WordPress.com', 'zgrab',
+		'curl/', 'StatusNet', 'OpenGraphReader/', 'Uptimebot/', 'python-opengraph-jaywink'];
+
 	if ($crawlerDetect->isCrawler()) {
+		foreach ($agents as $agent) {
+			if (stristr($_SERVER['HTTP_USER_AGENT'], $agent)) {
+				// @ToDo: Report every false positive here: https://github.com/JayBizzle/Crawler-Detect/issues/326
+				logger::notice('False positive', ['agent' => $_SERVER['HTTP_USER_AGENT']]);
```

Comment:
Please remove the logger call and the comment above, this isn't warranted. for this addon or for the upstream library.

Reply:
The logger calls is needed to be able to detect false positives. Depending upon the reaction to my issue we can see if reporting does make sense or not.